src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private int totalSupply = 0;
+    private int totalBuy = 0;
```

Comment:
If you are counting something in your method for example using some variable for the result, don’t make this variable a class field.
The class fields should represent the state of the class objects otherwise they should be local variables.